sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -342,7 +342,7 @@ internal static EntityLinkingTask ConvertToLinkedEntitiesTask(RecognizeLinkedEnt
                 Parameters = new EntityLinkingTaskParameters()
                 {
                     ModelVersion = action.ModelVersion,
-                    StringIndexType = action.StringIndexType,
+                    StringIndexType = StringIndexType.Utf16CodeUnit,
```

Comment:
NIT: Let's move `_stringIndexType` outside the client class file and use it here too.

Reply:
Created `Constants` class